media-libs/libpng/libpng-1.5.27.recipe

```diff
@@ -33,6 +33,10 @@ PROVIDES_devel="
 REQUIRES_devel="
 	libpng$secondaryArchSuffix == $portVersion base
 	"
+CONFLICTS_devel="
+	libpng12${secondaryArchSuffix}_devel
+	libpng16${secondaryArchSuffix}_devel
+	"
```

Comment:
It's not required: the conflicts are already declared on libpng1*_devel packages.